src/languages/en-US/system.json

```diff
@@ -29,15 +28,6 @@
 		"{{LOADING}} Tuning in to the right frequencies...",
 		"{{LOADING}} Reticulating splines..."
 	],
-	"messagePromptTimeout": "The prompt has timed out.",
-	"noResults": "I wasn't able to find any results for that query",
-	"parseError": "{{REDCROSS}} I failed to process the data I was given, sorry~!",
 	"prefixReminder": "The prefix in this server is set to: `{{prefix}}`",
-	"queryFail": "I am sorry, but the application could not resolve your request. Are you sure you wrote the name correctly?",
-	"textPromptAbortOptions": [
-		"abort",
-		"cancel",
-		"stop"
-	],
```

Comment:
all unused?

Reply:
We don't use `TextPrompt` anymore, but `ExceededLength`: https://github.com/skyra-project/skyra/blob/main/src/lib/util/Parsers/ExceededLength.ts